src/org/zaproxy/zap/extension/tokengen/TokenGenMessagesTableModel.java

```diff
@@ -1,10 +1,10 @@
 /*
  * Zed Attack Proxy (ZAP) and its related class files.
- * 
+ *
```

Comment:
This type of indentation/formatting changes are better if done in another commit, if you really want to do them. [1]

[1] https://github.com/zaproxy/zaproxy/wiki/DevGuidelines